bot/commands/utility/scheduleMessage.js

```diff
@@ -0,0 +1,66 @@
+const source = require('rfr');
+const moment = require('moment');
+
+const logger = source('bot/utils/logger');
+const { sendMessage } = source('bot/utils/util');
+const agenda = source('scheduler');
+
+// takes a crontab formatted string 6+ arguments, first
```

Comment:
I think this comment is cut off?

Reply:
fixed in 3b66bcf41c99fa3390cf7d0c2bca01618b295549